.github/workflows/dot-org-filter.yml

```diff
@@ -0,0 +1,41 @@
+name: Anaconda.org Isuse Notification to Slack
```

Comment:
```suggestion
name: Anaconda.org Issue Notification to Slack
```

Reply:
Also often good to add a couple lines of comments under this describing the workflow's purpose at a high level